resources.md

```diff
@@ -18,7 +18,7 @@ Tutorials
 * [Gemcutter & Jeweler](http://railscasts.com/episodes/183-gemcutter-jeweler)
 * [MicroGems: five minute RubyGems](http://jeffkreeftmeijer.com/2011/microgems-five-minute-rubygems/) - Gems so small that you can store them in a gist.
 * [Let's Write a Gem: Part 1](http://rakeroutes.com/blog/lets-write-a-gem-part-one/) and [Part 2](http://rakeroutes.com/blog/lets-write-a-gem-part-two/)
-* [Polishing Rubies](http://intridea.com/blog/tag/polishing%20rubies)
+* [Polishing Rubies](http://wayback.archive.org/web/20151009055337/http://www.intridea.com/blog/tag/polishing%20rubies)
```

Comment:
https for wayback worked for me.
![polishing rubies intridea blog 2015-12-08 11-34-08](https://cloud.githubusercontent.com/assets/118850/11661363/9bebbd32-9d9f-11e5-9796-715d53673b58.png)